src/stko/_internal/optimizers/openmm.py

```diff
@@ -16,6 +17,8 @@
 from stko._internal.utilities.exceptions import InputError
 from stko._internal.utilities.utilities import get_atom_distance
 
+warnings.filterwarnings("ignore", category=DeprecationWarning)
```

Comment:
why? this is a very general filter no (ie its module scope)

Reply:
Agreed, removed.